apps/Gruntfile.js

```diff
@@ -581,6 +585,8 @@ describe('entry tests', () => {
     'projects/index': './src/sites/studio/pages/projects/index.js',
     'report_abuse/report_abuse_form':
       './src/sites/studio/pages/report_abuse/report_abuse_form.js',
+    'reference_guides/index':
```

Comment:
Since this is an internal levelbuilder only page I think it should go in the `internalEntries`